i18n/it/cachyos_hello.ftl

```diff
@@ -0,0 +1,66 @@
+# About dialog
+about-dialog-title = CachyOS Hello
+about-dialog-comments = Schermata di benvenuto per CachyOS
+
+# Tweaks page
+tweaks = Modifiche
+fixes = Correzioni
+applications = Applicazioni
+removed-db-lock = Il blocco della base di dati di pacman è stato rimosso!
+lock-doesnt-exist = Il blocco della base di dati di pacman non esiste!
+orphans-not-found = Nessun pacchetto orfano trovato!
+package-not-installed = Il pacchetto '{$package_name}' non è stato installato!
+
+# Dns Connections page
+dns-settings = Impostazioni DNS
+select-connection = Seleziona Connessione:
+select-dns-server = Seleziona server DNS:
+apply = Applica
+reset = Reimposta
+dns-server-changed = Il server DNS è stato cambiato con successo!
+dns-server-failed = Impostazione del server DNS non riuscita!
+dns-server-reset = Il server DNS è stato reimpostato!
+dns-server-reset-failed = Reimpostazione del server DNS non riuscita!
+
+# Tweaks page (tweaks)
+tweak-enabled-title = {$tweak} abilitato
+
+# Tweaks page (fixes)
+remove-lock-title = Rimuovi il blocco della base di dati
+reinstall-title = Reinstalla tutti i pacchetti
+refresh-keyrings-title = Ricarica i keyring
+update-system-title = Aggiorna il sistema
+remove-orphans-title = Rimuovi gli orfani
+clear-pkgcache-title = Pulisci la cache dei pacchetti
+rankmirrors-title = Classifica i mirror
+dnsserver-title = Cambia server DNS
+
+# Main Page (buttons)
+button-about-tooltip = Informazioni
+button-web-resource-tooltip = Risorse in rete
+button-development-label = Sviluppo
+button-software-label = Software
+button-donate-label = Dona
```

Comment:
instead to use `dona` is more appropriate to use `donazioni`

Reply:
If I do that I'll have to change the source string too, otherwise it'd be innacurate

Reply:
it gonna be inaccurate anyway because you translate with dona but if you translate in italian donate it gonna be donare. So i don't think you need to change source string for use donazioni.

Reply:
Hmm, I think you misunderstood my translation there, since most buttons represent an action they are in imperative form, both in English and Italian, so "donate" maps to "dona", if it were "donazioni" on the Italian side, I'd have "donations" in English

Reply:
Good point!  
Perhaps that should reflect on the English one too for consistency? I'll leave that up to you though, I think "Supportaci" still works on its own so I'll change o it either way

Reply:
Nah, leave that don't reflect all translation very closely to english. Anyway yeah please replace with `Supportaci` looks more professional